app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -793,7 +796,7 @@ protected void onExternalAttachmentUpdated() {
         }
         String formStatus = formRecord.getStatus();
         if (FormRecord.STATUS_INCOMPLETE.equals(formStatus)) {
-            saveDataToDisk(false, false, null, true);
+            saveDataToDisk(false, false, null, true, true);
```

Comment:
not sure if this save is`userTriggered` as it happens in response to updating an attachment.

Reply:
true, good catch

Reply:
[019cd2d](https://github.com/dimagi/commcare-android/pull/2858/commits/019cd2d0521de8f6a49e3c3c1092fe59b0a49e38)